WhoIsThat/WhoIsThat.Android/Resources/Resource.designer.cs

```diff
@@ -2342,26 +2342,26 @@ public partial class Drawable
 			// aapt resource value: 0x7f020053
 			public const int avd_hide_password = 2130837587;
 			
-			// aapt resource value: 0x7f020132
-			public const int avd_hide_password_1 = 2130837810;
```

Comment:
The comment on this file states that any changes to it might cause unexpected behaviour.

You said to me yourself yesterday that something was not working and you left a comment that font might show errors. Personally, I'm not comfortable merging code if it generates errors, especially few hours before our first deadline. If you are worried that you will have nothing to show for lecturer, you can always just show him your local copy, and later on fix these errors without being in a rush.